### ISSUE TITLE
"jsonl" parser plugin has bundled in official package as "json"

### ISSUE THREAD

**Issue by kjmkznr** [takumakanari/embulk-parser-json#2]
https://github.com/embulk/embulk/blob/master/embulk-docs/src/release/release-0.8.5.rst

I can not use `embulk-parser-json`.

``` yaml
in:
  type: s3
  bucket: <bucket name>
  path_prefix: <path prefix>
  auth_method: profile
  profile_name: <profile>
  decoders:
  - {type: gzip}
  parser: 
    type: json
    root: $.Records
    schema:
      - {name: eventName, type: string}
      - {name: eventSource, type: string}
      - {name: awsRegion, type: string}
      - {name: sourceIPAddress, type: string}
      - {name: eventTime, type: string}
      - {name: requestID, type: string}
      - {name: eventID, type: string}
      - {name: userType, path: userIdentity.type, type: string}
      - {name: userArn, path: userIdentity.arn, type: string}
      - {name: userName, path: userIdentity.userName, type: string}
      - {name: accesskeyid, path: userIdentity.accessKeyId, type: string}
      - {name: userAgent, type: string}
      - {name: errorCode, type: string}
      - {name: errorMessage, type: string}
out: 
  type: file
  path_prefix: /tmp/cloudtrail/
  file_ext: json
```

``` bash
$ embulk preview cloudtrail.yml
org.embulk.spi.DataException: org.embulk.spi.json.JsonParseException: Failed to parse JSON: in
        at org.embulk.standards.JsonParserPlugin.run(org/embulk/standards/JsonParserPlugin.java:89)
        at org.embulk.spi.FileInputRunner.run(org/embulk/spi/FileInputRunner.java:153)
        at org.embulk.exec.PreviewExecutor$2$1.run(org/embulk/exec/PreviewExecutor.java:122)
        at org.embulk.spi.util.Filters$RecursiveControl.transaction(org/embulk/spi/util/Filters.java:96)
        at org.embulk.spi.util.Filters.transaction(org/embulk/spi/util/Filters.java:49)
        at org.embulk.exec.PreviewExecutor$2.run(org/embulk/exec/PreviewExecutor.java:111)
        at org.embulk.spi.FileInputRunner$RunnerControl$1$1.run(org/embulk/spi/FileInputRunner.java:123)
        at org.embulk.standards.JsonParserPlugin.transaction(org/embulk/standards/JsonParserPlugin.java:48)
        at org.embulk.spi.FileInputRunner$RunnerControl$1.run(org/embulk/spi/FileInputRunner.java:117)
        at org.embulk.spi.util.Decoders$RecursiveControl.transaction(org/embulk/spi/util/Decoders.java:77)
        at org.embulk.spi.util.Decoders$RecursiveControl$1.run(org/embulk/spi/util/Decoders.java:73)
        at org.embulk.standards.GzipFileDecoderPlugin.transaction(org/embulk/standards/GzipFileDecoderPlugin.java:30)
        at org.embulk.spi.util.Decoders$RecursiveControl.transaction(org/embulk/spi/util/Decoders.java:68)
        at org.embulk.spi.util.Decoders.transaction(org/embulk/spi/util/Decoders.java:33)
        at org.embulk.spi.FileInputRunner$RunnerControl.run(org/embulk/spi/FileInputRunner.java:114)
        at org.embulk.exec.BufferFileInputPlugin.transaction(org/embulk/exec/BufferFileInputPlugin.java:26)
        at org.embulk.spi.FileInputRunner.transaction(org/embulk/spi/FileInputRunner.java:65)
        at org.embulk.exec.PreviewExecutor.doPreview(org/embulk/exec/PreviewExecutor.java:108)
        at org.embulk.exec.PreviewExecutor.doPreview(org/embulk/exec/PreviewExecutor.java:98)
        at org.embulk.exec.PreviewExecutor.access$000(org/embulk/exec/PreviewExecutor.java:30)
        at org.embulk.exec.PreviewExecutor$1.run(org/embulk/exec/PreviewExecutor.java:70)
        at org.embulk.exec.PreviewExecutor$1.run(org/embulk/exec/PreviewExecutor.java:66)
        at org.embulk.spi.Exec.doWith(org/embulk/spi/Exec.java:25)
        at org.embulk.exec.PreviewExecutor.preview(org/embulk/exec/PreviewExecutor.java:66)
        at org.embulk.EmbulkEmbed.preview(org/embulk/EmbulkEmbed.java:169)
        at java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:498)
        at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(org/jruby/javasupport/JavaMethod.java:453)
        at org.jruby.javasupport.JavaMethod.invokeDirect(org/jruby/javasupport/JavaMethod.java:314)
```

**Comment by fs-wu** [comment]
same problem,and if I downgrade embulk to 0.8.4, it work, look like embulk use its own json plugin and the type name conflict.Can you change new type name?

**Comment by takumakanari** [comment]
@kjmkznr @fs-wu sorry for the very late reply. 

>  look like embulk use its own json plugin and the type name conflict

I see. 

I think it's better to change type name to another one, such as `jsonpath`, so I'll do that later.

Thanks for your report.

**Comment by takumakanari** [comment]
This issue was fixed in v0.0.4.

**Comment by kjmkznr** [comment]
Thanks!
